@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -879,8 +879,9 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
 
             const positionCount = (positions.length - basePosition) / 3;
             const count = indices.length - start;
+            const hasVertexColors = "vertexColors" in technique && technique.vertexColors === true;
 
-            if (isExtrudedPolygonTechnique(technique)) {
+            if (isExtrudedPolygonTechnique(technique) && hasVertexColors) {
```

Comment:
```suggestion
            if (isExtrudedPolygonTechnique(technique) && technique.vertexColors === true) {
```

Since ``isExtrudedPolygonTechnique`` is a typeguard.

Reply:
Excellent, thanks.